contracts/masset/Masset.sol

```diff
@@ -1,10 +1,14 @@
 pragma solidity 0.5.16;
 pragma experimental ABIEncoderV2;
 
+// TODO - remove
```

Comment:
I assume all this logging will be removed before merging.

Reply:
Yes, all `TODO`'s will be actioned and removed before deployment